kotest-assertions/kotest-assertions-json/src/commonMain/kotlin/io/kotest/assertions/json/compare.kt

```diff
@@ -377,23 +377,15 @@ private fun compareNumbers(
    }
 }
 
-private val fractionalZeroesRegex =
-   """(\.\d*)0+""".toRegex()
```

Comment:
this was moved, but note critically the `$` was added to the end to avoid `0.01` being stripped to `0.1` for example